core/trino-main/src/test/java/io/trino/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -563,7 +562,7 @@ public static PlanMatchPattern join(
                 new JoinMatcher(
                         joinType,
                         expectedEquiCriteria,
-                        expectedFilter.map(predicate -> rewriteIdentifiersToSymbolReferences(new SqlParser().createExpression(predicate, new ParsingOptions()))),
+                        expectedFilter.map(predicate -> PlanBuilder.expression(predicate)),
```

Comment:
using of `PlanBuilder.expression` in existing places should be separate commit

Reply:
done